prescient/simulator/reporting_manager.py

```diff
@@ -67,79 +67,98 @@ def setup_default_reporting(self, options, stats_manager: StatsManager):
     def setup_runtimes(self, options, stats_manager: StatsManager):
         runtime_path = os.path.join(options.output_directory, 'runtimes.csv')
         runtime_file = open(runtime_path, 'w', newline='')
-        runtime_columns = {"Date":       lambda hourly: str(hourly.date),
-                           "Hour":       lambda hourly: hourly.hour + 1,
-                           "Type":       lambda hourly: "SCED",
-                           "Solve Time": lambda hourly: hourly.sced_runtime}
-        runtime_writer = CsvReporter.from_dict(runtime_file, runtime_columns)
-        stats_manager.register_for_hourly_stats(runtime_writer.write_record)
+
+        ops_runtime_columns = {"Date":       lambda ops: str(ops.timestamp.date()),
+                               "Hour":       lambda ops: ops.timestamp.hour + 1,
```

Comment:
I wonder if we should just change this to ops.timestamp.hour -- I don't particularly like the "off-by-one" of this existing code, and with sub-hourly time-steps it's even more confusing.

Reply:
Maybe we do this as part of a separate PR, as this will break all our tests..